src/ResourceManager/KeyVault/Commands.KeyVault.Test/Scripts/PSHCommon/Common.ps1

```diff
@@ -13,51 +13,78 @@
 # ----------------------------------------------------------------------------------
```

Comment:
I'm not sure why you have captured a version of the Common.ps1 from the scenario tests common.  Seems like it woule be better to simply use the common ones.